ipinfo/cmd_login.go

```diff
@@ -67,3 +80,20 @@ func cmdLogin() error {
 
 	return nil
 }
+
+func checkToken(tok string) (bool, error) {
```

Comment:
Should rename to `isTokenReal` so it's clear what the returned `bool` means without the need for documentation.